app/tests/archives_tests/test_views.py

```diff
@@ -731,19 +732,17 @@ def test_archive_item_add_image(
                 method=client.post,
                 reverse_kwargs={
                     "pk": item.pk,
-                    "interface_slug": ci_img.slug,
-                    "archive_slug": archive.slug,
+                    "slug": archive.slug,
                 },
                 user=editor,
-                follow=True,
                 data={
                     ci_img.slug: image.pk,
                     f"WidgetChoice-{ci_img.slug}": WidgetChoices.IMAGE_SEARCH.name,
                 },
             )
-    assert response.status_code == 200
-    assert item.values.first().image.pk == image.pk
-    assert item.values.first() != old_civ
+    assert response.status_code == 302
+    assert item.values.filter(interface=ci_img).get().image.pk == image.pk
+    assert item.values.filter(interface=ci_img).get() != old_civ_img
```

Comment:
Watch out for this unnecessary pattern, seen it a few times now.

```suggestion
    assert item.values.get(interface=ci_img).image.pk == image.pk
    assert item.values.get(interface=ci_img) != old_civ_img
```